### ISSUE TITLE
Warnings on com.squareup.okhttp.OkHttpClient

### ISSUE THREAD

**Issue by liwang-pivotal** [instaclustr/cassandra-operator#37]
Operator keeps throwing out warnings like below periodically:
```
[OkHttp ConnectionPool] WARN  com.squareup.okhttp.OkHttpClient - A connection to https://192.168.99.101:8443/ was leaked. Did you forget to close a response body?
```

**Comment by liwang-pivotal** [comment]
Check if the latest K8s library is able to fix this issue.

**Comment by zegelin** [comment]
This could be because the `WatchService` isn't calling `close()` on the created `Watch`. Will add this to the PR.

**Comment by liwang-pivotal** [comment]
fixed by PR #47
